ods_ci/build/Dockerfile

```diff
@@ -38,31 +38,34 @@ WORKDIR $HOME/ods-ci
 RUN groupadd --gid 1001 ods-ci-users
 RUN useradd -r -u 1001 -g ods-ci-users ods-ci-runner
 
-COPY tests tests/
-COPY tasks tasks/
-COPY libs libs/
-COPY run_robot_test.sh  .
-COPY build/run.sh  build/run.sh
-COPY build/install_idp.sh  build/install_idp.sh
```

Comment:
I think this line got removed in this PR, but it should be present

Reply:
fixed